source/Octopus.Tentacle.Tests.Integration/TentacleCommandLineTests.cs

```diff
@@ -596,9 +596,6 @@ public async Task ShowConfigurationCommandLooksSensibleToHumans(TentacleConfigur
         {{
           ""Thumbprint"": ""{clientAndTentacle.Server.Thumbprint}"",
           ""CommunicationStyle"": 2,
-          ""KubernetesTentacleCommunicationMode"": {{
-            ""Value"": ""Polling""
-          }},
```

Comment:
This property only shows up on serialisation if the CommunicationStyle is `KubernetesTentacle` now.

Reply:
I did this because the example below is using `CommunicationStyle` of `Passive` but the `KubernetesTentacleCommunicationMode` says `Polling` which is confusing. I've just made it so that it's just not included at all to make it clearer.